include/tvm/auto_scheduler/measure.h

```diff
@@ -444,10 +469,9 @@ class RPCRunner : public ProgramRunner {
    * \param cooldown_interval The cool down interval between two measurements.
    * \param enable_cpu_cache_flush Whether to flush cache on CPU between repeated measurements.
    */
-  RPCRunner(const String& key, const String& host, int port, int priority, int n_parallel,
-            int timeout, int number, int repeat, int min_repeat_ms, double cooldown_interval,
-            bool enable_cpu_cache_flush);
-
+  RPCRunner(PackedFunc submit_func, const String& key, const String& host, int port, int priority,
```

Comment:
As previous comment, I doubt `PackedFunc` can not work well on the remote. Like `cache flush`, we use `string` to pass. See: https://github.com/apache/tvm/blob/main/python/tvm/auto_scheduler/measure.py#L849

Let us add test case to cover.

Reply:
This function is designed only for on host usage. PackedFunction is used here just to hold python object on native c++ level. There are no scenarios when this function will be transferred to device.